### DISCUSSION TITLE
Docs show wrong return type

### DISCUSSION THREAD

**Discussion by Filsommer** [fastapi/fastapi#10006] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
class Item(BaseModel):
    name: str
    price: float
    is_offer: bool

@app.get("/items/{item_id}")
def read_item() -> Tuple[Item, str]:
    return Item(name="name1", price=6.9, is_offer=False), "test_string"
```

### Description

When I specify the return type of my endpoint as a tuple of Item and string, the docs show the wrong return type.
[See issue in this screenshot](https://imgur.com/a/LjKMk6h)
Should be Item instead of string on the screenshot of the docs above.

Has anyone else had this issue?

### Operating System

macOS

### FastAPI Version

0.100.1

### Pydantic Version

1.10.8

### Python Version

Python 3.9.10

**Comment by YuriiMotov** [comment]
Duplicate: https://github.com/fastapi/fastapi/discussions/13579
